x11-drivers/nvidia-drivers/nvidia-drivers-525.85.12.ebuild

```diff
@@ -16,13 +16,12 @@ IUSE=""
 S="${WORKDIR}"
 
 RDEPEND="
-	=x11-drivers/nvidia-metadata-${PV}
+  =x11-drivers/nvidia-metadata-${PV}
```

Comment:
I think we can move the contents of `nvidia-metadata` ebuild into this `nvidia-drivers` ebuild here to have all at one place now.

Reply:
Maybe let's only focus on this here and then merge? Assuming that the driver still works.